Externals/rangeset/include/rangeset/rangesizeset.h

```diff
@@ -322,15 +322,15 @@ template <typename T> class RangeSizeSet {
     return get_from(it) <= value && value < get_to(it);
   }
 
-  size_t size() const {
+  std::size_t size() const {
```

Comment:
Could you include `<cstddef>` and fix rangeset.h as well?

Reply:
Good points! Added include and tweaked `rangeset.h`